src/lib/drishti/core/Field.h

```diff
@@ -24,16 +24,20 @@ DRISHTI_CORE_NAMESPACE_BEGIN
 template <typename T>
 struct Field
 {
-    Field()
-         
-    = default;
+    Field() = default;
```

Comment:
Eventually we can use [optiona](https://en.cppreference.com/w/cpp/utility/optional) for this (C++17).